### PR TITLE
[hotfix][examples] Add zookeeper.connect to usage line

### PR DESCRIPTION
**Pull request by stefanobaghino**
Follow-up to [this thread](http://apache-flink-user-mailing-list-archive.2336050.n4.nabble.com/Unable-to-Read-from-Kafka-zookeeper-connect-error-td5577.html) on the Flink user mailing list.

### CODE REVIEW

File: flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/kafka/WriteIntoKafka.java [hunk 1]
```diff
@@ -38,7 +38,8 @@
 	public static void main(String[] args) throws Exception {
 		ParameterTool parameterTool = ParameterTool.fromArgs(args);
 		if(parameterTool.getNumberOfParameters() < 2) {
```
Comment (rmetzger) [inline flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/kafka/WriteIntoKafka.java]: I guess the number of expected parameters is 3 now.

Comment (stefanobaghino) [inline flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/kafka/WriteIntoKafka.java]: Right, thanks!

**Comment by rmetzger** [comment]
Merging ...

**Comment by rmetzger** [comment]
Can you close the issue manually? I forgot to include the close message into the commit

**Comment by rmetzger** [comment]
Just noticed that we added the zookeeper.connect to the wrong file. This belongs to the consumer not the producer.
Will fix it.

**Comment by stefanobaghino** [comment]
You're right, sorry. I can fix this right now.

**Comment by rmetzger** [comment]
Its fine, I fixed it already and will push a hotfix.

**Comment by stefanobaghino** [comment]
Ok, thanks!

**Pull request closed without merging**